app/src/main/java/com/example/android/codelabs/paging/data/GithubRepository.kt

```diff
@@ -51,12 +53,17 @@ class GithubRepository(private val service: GithubService) {
      * Search repositories whose names match the query, exposed as a stream of data that will emit
      * every time we get more data from the network.
      */
-    suspend fun getSearchResultStream(query: String): Flow<RepoSearchResult> {
+    suspend fun getSearchResultStream(query: String): Flow<PagingData<Repo>> {
         Log.d("GithubRepository", "New query: $query")
         lastRequestedPage = 1
         requestAndSaveData(query)
```

Comment:
can you remove all of this requestAndSaveData stuff?

should this entire class just be the call to PagingDataFlow?

Reply:
yes. and cleaned up more unused methods afterwards.